cmd/kube-controller-manager/app/core_test.go

```diff
@@ -141,3 +143,175 @@ func TestController_DiscoveryError(t *testing.T) {
 		}
 	}
 }
+
+func TestSetNodeCIDRMaskSizes(t *testing.T) {
+	_, dummyIPv4CIDR, err := net.ParseCIDR("1.2.3.0/32")
```

Comment:
nit, a /32 mask represents a host, it may better use a /16 or a more representative mask, at least greater than /24 that is the default node mask